setup.py

```diff
@@ -88,7 +75,7 @@ def find_rel_test():
     license="GPLv2",
     keywords="software build building installation installing compilation HPC scientific",
     url="https://easybuild.io",
-    packages=easybuild_packages,
+    packages=find_packages(),
```

Comment:
I will need to check this thoroughly, since I don't want to me up a release due to this change...